scripts/release/params.py

```diff
@@ -0,0 +1,94 @@
+import sys
+
+import requests
+
+QUAY_REPOS = {
+    "IMAGES_APISERVER":  "ds-pipelines-api-server",
+    "IMAGES_ARTIFACT":  "ds-pipelines-artifact-manager",
+    "IMAGES_PERSISTENTAGENT":  "ds-pipelines-persistenceagent",
+    "IMAGES_SCHEDULEDWORKFLOW":  "ds-pipelines-scheduledworkflow",
+    "IMAGES_MLMDENVOY":  "ds-pipelines-metadata-envoy",
+    "IMAGES_MLMDGRPC":  "ds-pipelines-metadata-grpc",
+    "IMAGES_MLMDWRITER":  "ds-pipelines-metadata-writer",
+    "IMAGES_DSPO":  "data-science-pipelines-operator",
+}
+
+ARCH = "amd64"
+
+
+def fetch_quay_repo_tag_digest(quay_repo, quay_org, tag):
+    api_url = "https://quay.io/api/v1/repository/{0}/{1}/tag/?specificTag={2}".format(
+        quay_org,
+        quay_repo,
+        tag
+    )
+    response = requests.get(api_url).json()
+    tags = response['tags']
+
+    if len(tags) == 0 or 'end_ts' in tags[0]:
+        print("Tag does not exist or was deleted.", file=sys.stderr)
+        exit(1)
+    return tags[0]['manifest_digest']
+
+
+def fetch_rh_repo_tag_digest(repo, tag):
+    registry = "registry.access.redhat.com"
+    api_url = "https://catalog.redhat.com/api/containers/v1/repositories/registry/{0}/repository/{1}/tag/{2}".format(
+        registry,
+        repo,
+        tag
+    )
+    response = requests.get(api_url).json()
+
+    amd_img = {}
+    for img in response['data']:
+        if img['architecture'] == 'amd64':
```

Comment:
what if `architecture` doesn't exist or stops existing?

Reply:
That would be an api change on rh registry's part, which would probably break...a lot of things if they did that. But I added a check regardless.

Reply:
you added the check, might as well use the cleaner syntax

```suggestion
        if arch == 'amd64':
```